packages/mobile-app/.eslintrc.js

```diff
@@ -4,5 +4,6 @@ module.exports = {
   plugins: ["prettier"],
   rules: {
     "prettier/prettier": "error",
+    "import/no-unresolved": "off",
```

Comment:
I can't figure out how to set up path aliases while also making ESLint happy

Reply:
Only suggestion would be to add a comment for why we have it

Reply:
Just fixed it hah